%!TEX TS-program = lualatex
%!TEX encoding = UTF-8 Unicode

\documentclass[11pt]{article}
\usepackage{graphicx}
	\graphicspath{{/Users/goby/Pictures/teach/300/}} % set of paths to search for images

\usepackage{geometry}
\geometry{letterpaper, bottom=1in}                   
%\geometry{landscape}                % Activate for for rotated page geometry
%\usepackage[parfill]{parskip}    % Activate to begin paragraphs with an empty line rather than an indent
\usepackage{amssymb, amsmath}
\usepackage{mathtools}
	\everymath{\displaystyle}

\usepackage{fontspec}
\setmainfont[Ligatures={TeX}, BoldFont={* Bold}, ItalicFont={* Italic}, BoldItalicFont={* BoldItalic}, Numbers={Proportional}]{Linux Libertine O}
\setsansfont[Scale=MatchLowercase,Ligatures=TeX]{Linux Biolinum O}
%\setmonofont[Scale=MatchLowercase]{Inconsolata}
\usepackage{microtype}

\usepackage{unicode-math}
\setmathfont[Scale=MatchLowercase]{Asana Math}
%\setmathfont[Scale=MatchLowercase]{XITS Math}

% To define fonts for particular uses within a document. For example, 
% This sets the Libertine font to use tabular number format for tables.
\newfontfamily{\tablenumbers}[Numbers={Monospaced}]{Linux Libertine O}
\newfontfamily{\libertinedisplay}{Linux Libertine Display O}

\usepackage{booktabs}
%\usepackage{tabularx}
\usepackage{longtable}
%\usepackage{siunitx}
\usepackage{array}
\newcolumntype{L}[1]{>{\raggedright\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}

\usepackage{enumitem}
\usepackage{hyperref}
%\usepackage{placeins} %PRovides \FloatBarrier to flush all floats before a certain point.
\usepackage{hanging}

\usepackage{titling}
\setlength{\droptitle}{-60pt}
\posttitle{\par\end{center}}
\predate{}\postdate{}

%\setlength\parindent{0pt}

\usepackage{fancyhdr}
\headheight = 13.6pt
\fancypagestyle{firstpage}{%
	\fancyhf{}
	\fancyhead[L]{BI 300: Evolution}
	\fancyhead[R]{Name: \enspace \makebox[2.5in]{\hrulefill}}
	\renewcommand{\headrulewidth}{0pt}
}

\fancyhf{}
\pagestyle{fancy}
\fancyhead[R]{\thepage}
\renewcommand{\headrulewidth}{0.4pt}

\setlength\parindent{0pt}



\begin{document}
\thispagestyle{empty}
\subsection*{Hardy-Weinberg Vocabulary}

You are responsible for learning the meaning of these terms. They'll be used in forthcoming exercises and will appear on the exams.\medskip

\textbf{Census Population Size (\emph{N})}: The total number of
individuals in a population. Compare to effective population size.\medskip

\textbf{Effective Population Size (\emph{N}\textsubscript{e})}: The
number of breeding individuals in a population. Compare to census
population size. In this course, ``population size'' refers to
effective population size unless noted specifically.\medskip

\textbf{Gene Flow}: The movement of alleles between populations. Not all
gene flow is associated with migration of individuals. For example, wind-borne pollen
dispersing among populations represents gene flow without migration.\medskip

\textbf{Genetic Drift}: The random variation in haplotype frequencies
due to random events.\medskip

\textbf{Haplotype}: Any distinct sequence of DNA. May be coding or
non-coding.\medskip

\textbf{Heterozygosity}: The proportion of individuals in a population
that are heterozygous at a locus. Can be used for two or more loci.\medskip

\textbf{Heterozygote Advantage}: Higher relative fitness (see) of
heterozygotes compared to either homozygote. Also known as
heterosis or overdominance.\medskip

\textbf{Heterozygote Disadvantage}: Lower relative fitness (see) of
heterozygotes compared to either homozygote. Also known as
underdominance.\medskip

\textbf{Homozygosity}: The proportion of individuals in a population
that are homozygous at a locus. Can be used for two or more loci.\medskip

\textbf{Migration Rate (\emph{m})}: The rate at which individuals
disperse between populations. Migration of individuals causes gene flow.\medskip

\textbf{Mutation Rate (Âµ)}: The rate at which new mutations arise in a
population. A source of new genetic variation. Also represents the rate
at which new haplotypes substitute for previous haplotypes in a
population (see Substitution).\medskip

\textbf{Negative Selection (Purifying Selection)}: Any form of natural
selection that decreases the frequencey of detrimental alleles in a
population. $\frac{dN}{dS} < 1$\medskip

\textbf{Neutral Mutation}: Any mutation that is not subject to natural
selection, such as a point mutation in a non-coding region or at a
synonymous third codon position.\medskip

\textbf{Nonsynonymous Substitutions}: any nucleotide mutation that
changes the amino acid sequence in a protein. The number of
nonsynonymous substitutions is represented by $dN$.\medskip

\textbf{Population Size (\emph{N})}: The total number of individuals in
a population. Often synonymous with census population size, depending
on context. In this course, ``population size'' refers to
effective population size unless noted specifically.  Compare to effective population size.\medskip

\textbf{Positive Selection}: Any form of natural selection that
increases the frequencey of beneficial alleles in a population. $\frac{dN}{dS} > 1$\medskip

\textbf{Relative Fitness (\emph{w})}: The fitness of a genotype relative
to the fitness of other genotypes in a population. Usually set to 1 for
the most fit genotype in the population, although this can vary among
texts.\medskip

\textbf{Selection Coefficient (\emph{s})}: The difference in relative
fitness between a genotype and the most fit genotype, calculated as
\emph{s} = 1-\emph{w}. For example, the selection coefficient for a
genotype with a relative fitness of 0.95 is 0.05.\medskip

\textbf{Strength of Selection}: see Selection Coefficient.\medskip

\textbf{Synonymous Substitution}: any nucleotide mutation that does not
change the amino acid sequence in a protein. The number of synonymous
substitutions is represented by \emph{dS.}\medskip

\textbf{Substitution (\emph{k})}: The replacement of an existing
haplotype by a new haplotype in a population. For example, haplotype
\emph{A} is replaced by haplotype \emph{a} by genetic drift in a small
population. Haplotype \emph{a} substituted for haplotype \emph{A}. Do
not confuse with nonsynonymous and synonymous amino acid substitutions
in proteins.\medskip

\textbf{Modes of Selection}\medskip

The operation of natural selection, or mode of selection, in a
population can be described by comparing the relative fitness of the
genotypes in a population. Assume a population has two haplotypes,
\emph{A}\textsubscript{1} and \emph{A}\textsubscript{2}. In the
following abstract definitions, subscripts 1 and 2 represent the
\emph{A}\textsubscript{1} and \emph{A}\textsubscript{2} haplotypes,
respectively. Thus, \emph{w}\textsubscript{12} is the relative fitness
of \emph{A}\textsubscript{1}\emph{A}\textsubscript{2} heterozygotes in
the population.\medskip

\textbf{Directional Selection}: \emph{w}\textsubscript{11}
\textgreater{} \emph{w}\textsubscript{12} \textgreater{}
\emph{w}\textsubscript{22}

\qquad The population will shift towards the
\emph{A}\textsubscript{1}\emph{A}\textsubscript{1} phenotype because
\emph{w}\textsubscript{11} has the greatest fitness.\medskip

\textbf{Stabilizing Selection}: \emph{w}\textsubscript{11}\textless{}
\emph{w}\textsubscript{12} \textgreater{} \emph{w}\textsubscript{22}

\qquad The population will maintain the heterozygous phenotype at high
frequency because \emph{w}\textsubscript{12} has the highest fitness.
Both haplotypes will be maintained in the population. See Heterozygote
Advantage.\medskip

\textbf{Disruptive Selection}: \emph{w}\textsubscript{11} \textgreater{}
\emph{w}\textsubscript{12} \textless{} \emph{w}\textsubscript{22}

\qquad The population will shift towards the two homozygous phenotypes. The
relative frequency of each homozygous phenotype will depend on the
relative fitness difference between \emph{w}\textsubscript{11} and
\emph{w}\textsubscript{22}. See Heterozygote Disadvantage.\medskip

\textbf{A few basics of population and molecular genetics.}

$\frac{1}{2N_e} =$ the probability or frequency at which a new mutation will drift to
fixation in a population.\medskip

$2N_e\mu =$ the number of new mutations that will
occur, on average, in a population each generation.\medskip

$k = \frac{1}{2N_e} \times 2N_e\mu = \mu =$ the average rate at which new haplotypes substitute for previously
existing haplotypes in a population by genetic drift.\medskip

If $s \ll \frac{1}{4N_e}$, then drift will be the primary determinant of haplotype frequency
change.\medskip

If $s \gg \frac{1}{4N_e}$, then selection will be the primary determinant of haplotype
frequency change.\medskip

if $\frac{dN}{dS} \ll 1$, then the allele is evolving by negative (purifying) selection.
Statistical tests are used to determine if the ratio is significantly
different from or statistically equal to 1.\medskip

if $\frac{dN}{dS} \gg 1$, then the allele is evolving by positive selection. Statistical
tests are used to determine if the ratio is significantly different from
or statistically equal to 1.\medskip

if $\frac{dN}{dS} = 1$, then the haplotype is evolving by genetic drift. Statistical tests
are used to determine if the ratio is significantly different from or
statistically equal to 1.\medskip

\end{document}  