%!TEX TS-program = lualatex
%!TEX encoding = UTF-8 Unicode

\documentclass[t]{beamer}

%%%% HANDOUTS For online Uncomment the following four lines for handout
%\documentclass[t,handout]{beamer}  %Use this for handouts.
%\usepackage{handoutWithNotes}
%\includeonlylecture{student}
%\pgfpagesuselayout{3 on 1 with notes}[letterpaper,border shrink=5mm]
%	\setbeamercolor{background canvas}{bg=black!5}
\usefonttheme{professionalfonts}


%%% Including only some slides for students.
%%% Uncomment the following line. For the slides,
%%% use the labels shown below the command.

%% For students, use \lecture{student}{student}
%% For mine, use \lecture{instructor}{instructor}

% FONTS
\usepackage{fontspec}
\def\mainfont{Linux Biolinum O}
\setmainfont[Ligatures={Common, TeX}, Contextuals={NoAlternate}, BoldFont={* Bold}, ItalicFont={* Italic}, Numbers={Proportional, OldStyle}]{\mainfont}
%\setmonofont[Scale=MatchLowercase]{Inconsolata} 
\setsansfont[Scale=MatchLowercase]{Linux Biolinum O} 
\setmonofont{Linux Libertine Mono O}

\usepackage{microtype}

\usepackage{unicode-math}
\setmathfont[Scale=MatchLowercase]{Asana Math}

\usepackage{graphicx}
	\graphicspath{%
	{/Users/goby/Pictures/teach/300/lectures/}%
	{/Users/goby/Pictures/teach/163/common/}} % set of paths to search for images

\usepackage{amsmath,amssymb}

%\usepackage{units}

\usepackage{booktabs}
\usepackage{array}
\newcolumntype{L}[1]{>{\raggedright\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}


\usepackage{multicol}
%	\setlength{\columnsep=1em}
\usepackage{enumitem}
\usepackage{textcomp}
\usepackage{setspace}
\usepackage{tikz}
	\tikzstyle{every picture}+=[remember picture,overlay]

\mode<presentation>
{
  \usetheme{Lecture}
  \setbeamercovered{invisible}
  \setbeamertemplate{items}[square]
}

\usepackage{calc}
\usepackage{hyperref}

\newcommand\HiddenWord[1]{%
	\alt<handout>{\rule{\widthof{#1}}{\fboxrule}}{#1}%
}


\usepackage{xifthen}
\newcommand{\futuyma}[1]{%
	\ifthenelse{\isempty{#1}}%
	{Futuyma \& Kirkpatrick 2017, 4th ed.}%
	{Fig.~#1~Futuyma \& Kirkpatrick 2017, 4th ed.}%
}

% This defines \amper for the fancy ampersand
% to be used in the header. See
% https://tex.stackexchange.com/a/58185/39194
\usepackage{xspace}
\newfontfamily\amperfont[Style=Alternate]{Linux Libertine O}    
\makeatletter
\DeclareRobustCommand{\amper}{{\amperfont\ifx\f@shape\scname\smaller[1.2]\fi\&}\xspace}
\makeatother

\newcommand{\backskip}{\vspace{-0.5\baselineskip}}

\begin{document}

\lecture{student}{student}

\begin{frame}[t]{Drift causes \highlight{non-adaptive evolution.}}
\centering

\includegraphics[width=0.97\linewidth]{futuyma_fig7-4}
\end{frame}

%%

\begin{frame}{Gene trees can evolve by drift alone.}
\backskip

\begin{multicols}{2}
\noindent \includegraphics[width=\linewidth]{futuyma_fig7-5a}

\columnbreak

\pause
\noindent \includegraphics[width=\linewidth]{futuyma_fig7-5b}

\end{multicols}

\vfilll

\tinyfill \futuyma{7.5}

\end{frame}

%%

\begin{frame}{The probability of fixation equals the frequency of the haplotype.}
\backskip

\begin{multicols}{2}

\onslide<1-4>
\hangpara What is the probability that a \emph{new} mutation in effective population of 50 diploid individuals drifts to fixation?

\mode<beamer>{%
\onslide<2-4>
\hangpara $N_e = 50.$

\onslide<3-4>
\hangpara $2N_e = 100$ copies of the locus.

\onslide<4>
\hangpara $1$ new mutation out of $2N$ copies.

\begin{equation*}
\color{orange5}{\dfrac{1}{2N_e}}
\end{equation*}
}


\columnbreak

\onslide<1-4>
\noindent \includegraphics[width=\linewidth]{futuyma_fig7-5b}


\end{multicols}

\vfilll

\tinyfill \futuyma{7.5}

\end{frame}


%%


%\begin{frame}{Non-adaptive evolution by drift occurs because$\dots$}
%
%\backskip
%\begin{multicols}{2}
%\hangpara Haplotype frequencies fluctuate randomly,
%
%\hangpara One haplotype drifts to fixation ($p = 1.0$), and
%
%\hangpara New haplotypes replace old haplotypes.
%
%\columnbreak
%
%\centering
%\noindent\includegraphics[width=0.8\linewidth]{drift_haplotype_fluctuation}
%
%\bigskip
%
%\noindent\includegraphics[width=0.8\linewidth]{drift_haplotype_substitution}
%
%\end{multicols}
%
%\vfilll
%
%\tiny \textcopyright\,Pearson Prentice Hall, 2004.
%\end{frame}

%%

\begin{frame}{New neutral mutations occur at rate $\mu.$}

\backskip

\centering

\includegraphics[height=0.85\textheight]{drift_new_neutral_mutation}

\vfilll

\tinyfill Modified from \futuyma{7.5}

\end{frame}

\begin{frame}{Over time, new haplotypes replace old haplotypes.}

\vspace{-0.5\baselineskip}

\centering

\includegraphics[height=0.85\textheight]{drift_haplotype_substitution}

\vfilll

\tinyfill \textcopyright\,Pearson Prentice Hall, 2004.
 
\end{frame}

%%

\begin{frame}{Haplotype substitution occurs at the \highlight{neutral mutation rate.}}

\backskip

\hangpara $\mu$ = neutral mutation rate / haplotype / generation.

\hangpara $2N_e$ haplotype copies in diploid population.

\hangpara $2N_e\mu$ new mutations each generation (avg).

\hangpara $\dfrac{1}{2N_e}$ = fixation probability of each new mutation.

\hangpara Average rate of haplotype substitution is

\begin{equation*}
2N_e\mu \times \dfrac{1}{2N_e} = \mu
\end{equation*}
\end{frame}


%%

\begin{frame}{The \highlight{neutral theory of molecular evolution} states that molecular evolution is neutral.}

\backskip

\begin{multicols}{2}
\hangpara D\textsc{na} evolves by genetic drift at  rate equal to the neutral mutation rate $\mu$ in the population.

\hangpara The rate of molecular evolution is greatest at \textsc{dna} positions \emph{least} like to affect function.

\hangpara Purifying selection eliminates deleterious mutations.

\columnbreak

\noindent \includegraphics[width=\linewidth]{futuyma_fig7-19}

\end{multicols}

\vfilll

\tinyfill \futuyma{7.19}

\end{frame}

%%

\begin{frame}{Can drift affect frequencies of alleles under selection?}

\backskip

\centering

\includegraphics[height=0.85\textheight]{futuyma_fig7-13}

\vfilll

\tinyfill \futuyma{7.13}

\end{frame}

\lecture{instructor}{instructor}

\begin{frame}{Time for some simulations.}
\hangpara $s = 0.01$ for each copy of the $A_1$ allele.

\hangpara If $w_{11}$ has maximum fitness of 1.0, then

\hangpara $w_{12} = 0.99$ and $w_{22} = 0.98.$

\hangpara $2N_e$ will reduce from 5000 to 5.

\hangpara Starting frequency of $A_1 = 0.25$. Recall probability of fixation by drift alone equals frequency of allele.

\hangpara Let's start with no selection.


\end{frame}

\lecture{student}{student}

\begin{frame}
\begin{multicols}{2}
\hangpara If $s\gg\frac{1}{2N_e}$ then selection will tend to overcome drift.

\hangpara If $s\ll\frac{1}{2N_e}$ then drift will tend to overcome selection.

\hangpara What are the implications for population size on evolution of alleles with very small fitness effects (say, $10^{-5}$)?

\columnbreak

\noindent \includegraphics[width=\linewidth]{drift_selection_drift_balance}

\end{multicols}

\vfilll

\tinyfill Modified from \futuyma{7.13}

\end{frame}


%%

\begin{frame}{Alleles with very small fitness effects evolve as if they were selectively neutral unless $N_e$ is very large.}

\backskip

\begin{multicols}{2}
\noindent\includegraphics[width=0.9\linewidth]{futuyma_fig7-14b}

\columnbreak

\noindent\includegraphics[width=0.9\linewidth]{futuyma_fig7-14c}
\end{multicols}

Gray: $2N_es \rightarrow 0$

\vfilll

\tinyfill \futuyma{7.14}

\end{frame}


%%

\begin{frame}{Does selection occur at the molecular level?}

\hangpara \highlight{Selective sweep:} genetic hitchhiking.

\hangpara \highlight{Positive selection:} More non-synonymous than synonymous substitutions in a gene.

\hangpara \highlight{Codon bias:} Non-random use of codons.

\end{frame}

%%

\begin{frame}{$dN > dS$ is genetic evidence of \highlight{positive selection.}}

\backskip

\noindent \includegraphics[width=\linewidth]{drift_dn_ds}

\vfilll

\tinyfill \textcopyright\,Zimmer and Emlen.

\end{frame}

%%

\begin{frame}{Snake venoms evolve under positive diversifying selection.}

\backskip

\includegraphics[height=0.7\textheight]{drift_snake_venoms}

\begin{tikzpicture}
\node at (8.7,1) [text width=5.5cm] {\small Disintegrin proteins from rattlesnakes and vipers. Blue and red cirles show amino acid replacements from positive selection.};
\end{tikzpicture}

\vfilll

\tiny Juarez et al.~2008. Mol.~Biol.~Evol.~25: 2391

\end{frame}

%%

\begin{frame}{Snake venoms evolve under positive diversifying selection.}

\vspace{-\baselineskip}

\centering
\includegraphics[height=0.78\textheight]{drift_snake_venom_ratios}


\vfilll

\tinyfill Juarez et al.~2008. Mol.~Biol.~Evol.~25: 2391


\end{frame}

%%

\begin{frame}{\textit{FOXP2} evolved under positive directional selection.}

\backskip

\centering

\includegraphics[width=\linewidth]{drift_foxp2}


\vfilll

\tinyfill After Engard et al.~2002. Nature 418: 869.

\end{frame}

%%

\begin{frame}{\highlight{Codon bias} is the non-random use of codons.}
\backskip

\noindent \includegraphics[width=\linewidth]{drift_codon_bias}\\
\raisebox{-0.7ex}{\Large *}\,= underused codons.

\vfilll

\tinyfill Powell and Moriyama 1997. P\textsc{nas} 94: 7784.
\end{frame}
\end{document}
