%!TEX TS-program = lualatex
%!TEX encoding = UTF-8 Unicode

\documentclass[11pt, addpoints]{exam}
\usepackage{graphicx}
	\graphicspath{{/Users/goby/Pictures/teach/300/}
		{/Users/goby/Pictures/teach/300/exercises/}
	{img/}} % set of paths to search for images

\usepackage{geometry}
\geometry{letterpaper, bottom=1in}                   
%\geometry{landscape}                % Activate for for rotated page geometry
\usepackage[parfill]{parskip}    % Activate to begin paragraphs with an empty line rather than an indent
\usepackage{amssymb, amsmath}
\usepackage{mathtools}
	\everymath{\displaystyle}

\usepackage{fontspec}
\setmainfont[Ligatures={TeX}, BoldFont={* Bold}, ItalicFont={* Italic}, BoldItalicFont={* BoldItalic}, Numbers={Proportional, OldStyle}]{Linux Libertine O}
\setsansfont[Scale=MatchLowercase,Ligatures=TeX]{Linux Biolinum O}
\setmonofont[Scale=MatchLowercase]{Inconsolatazi4}
\usepackage{microtype}

\usepackage{unicode-math}
\setmathfont[Scale=MatchLowercase]{Asana Math}

\newfontfamily{\tablenumbers}[Numbers={Monospaced}]{Linux Libertine O}
\newfontfamily{\libertinedisplay}{Linux Libertine Display O}

\usepackage{booktabs}
%\usepackage{tabularx}
%\usepackage{longtable}
%\usepackage{siunitx}

\usepackage{caption}
	\captionsetup{labelsep=period} % Removes colon following figure / table number.

\usepackage{array}
\newcolumntype{L}[1]{>{\raggedright\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\\arraybackslash\hspace{0pt}}p{#1}}

\usepackage{enumitem}
\setlist{resume, label=\textsc{\alph*}., leftmargin=*}
\setlist[1]{labelindent=\parindent}

\usepackage[sc]{titlesec}

%\usepackage{titling}
%\setlength{\droptitle}{-60pt}
%\posttitle{\par\end{center}}
%\predate{}\postdate{}

\renewcommand{\solutiontitle}{\noindent}
\unframedsolutions
\SolutionEmphasis{\bfseries}

\renewcommand{\questionshook}{%
	\setlength{\leftmargin}{-\leftskip}%
}

\pagestyle{headandfoot}
\firstpageheader{\textsc{bi} 300: Evolution}{}{\ifprintanswers\textbf{KEY} \else Name \rule{2.5in}{0.4pt}\fi}
\runningheader{}{}{\footnotesize{pg. \thepage}}
\footer{}{}{}
\runningheadrule

%\printanswers

\begin{document}

\subsection*{Giraffes: one species or many? Part 1. (\numpoints\ points)}

Giraffes are found in several widely separated areas across western,
central and southern Africa. The areas are not separated by obvious
geographic barriers and may be remnants of a once wider distribution
across the entire region (Fig.~\ref{fig:giraffes}, left panel). As a result, most
researchers have treated the giraffe as a single species, \textit{Giraffa
camelopardalis}. Confusing the issue, however, is that the coat patterns
on giraffes are highly variable (Fig.~1, right panel) although each
pattern tends to be associated with a specific region. Some researchers
have argued that the giraffe is a \emph{single} species with several
subspecies. Others have argued that coat patterns are a distinguishing
feature for \emph{different} species of giraffes.

\begin{center}
	\includegraphics[width=\textwidth]{giraffe_fig1_color}
	\captionof{figure}{The distribution of African giraffes (left panel; dark areas) and the coat patterns of giraffes from different areas (right panel).\label{fig:giraffes}}
\end{center}

A study by David Brown and colleagues\footnote{Brown,~D.\,M.~et al.~2007.
  Extensive population genetic structure in the giraffe. BMC Biology 5:
  57.} used a variety of phylogenetic and population genetic analyses to
estimate the actual number of giraffe species. You will perform one part
of their analysis using their mitochondrial \textsc{dna} sequences. They used
1707 nucleotides sampled from a protein-encoding gene called cytochrome
\textit{b} and part of a non-coding region called the control region. The
sequences were taken from 35 individuals from six different regions with
five different coat patterns (Fig.~\ref{fig:sample_sites}). Each sequence is identified with
the pattern of the giraffe (e.g., pattern1, pattern2, \ldots{},
pattern5), plus a unique haplotype identifier (h1, h2, \ldots{}, h35).

You will perform your analysis at a website that will align your
sequences and build a phylogenetic tree for you. You will then interpret
the tree and decide whether you would consider the giraffe to be a
single species or if you would recognize some number of different
species. Finally, answer the three questions on page \pageref{sec:questions}. Type your
answers to the questions in a Word document and upload a copy to the dropbox by the assigned due date.

\begin{center}
	\includegraphics[width=0.75\textwidth]{giraffe_fig2_samples}
	\captionof{figure}{Sampled sites and coat patterns for this study. The numbers
correspond to the pattern numbers that identify the \textsc{dna} sequences (Brown~et~al.~2007).\label{fig:sample_sites}}
\end{center}

\subsection*{Perform the phylogenetic analysis}

The website you will use performs several steps for you automatically,
based on settings you provide (given to you below).

\begin{enumerate}
\item
  Go to the course website. Click on the link called “Giraffe \textsc{dna}
  Sequences.” Select all and copy the sequences.
\item
  Click on the link that says “Phylogenetic Analysis Website.” This
  will take you to the website where you will perform the analysis (Fig.
  \ref{fig:home_page}). Scroll down just a bit until you see the link called “Advanced,”
  then click it.
\item Create your workflow (Fig.~\ref{fig:workflow}). The workflow consists of several steps
  that you need to obtain your final tree. The workflow will align your
  sequences,  perform a phylogenetic analysis, and draw the
  tree. You do not need your alignment curated so uncheck Gblocks and
  then click the “Create Workflow” button.
\item
  Paste the giraffe sequences that you copied earlier into the area
  marked “Input Data” (Fig.~\ref{fig:paste_sequences}) and then scroll down to set the
  parameters for the PhyML phylogenetic analysis (Fig.~\ref{fig:phyml}).
\end{enumerate}

\begin{center}
	\includegraphics[width=0.7\textwidth]{fig3_home_page}
	\captionof{figure}{Home page of Phylogeny.fr, an online site for performing phylogenetic analyses.\label{fig:home_page}}
\end{center}

\vfill

\begin{center}
	\includegraphics[width=0.6\textwidth]{fig4_workflow}
	\captionof{figure}{Workflow settings.\label{fig:workflow}}
\end{center}

\newpage

\begin{center}
	\includegraphics[width=0.7\textwidth]{fig5_paste_sequences}
	\captionof{figure}{Input data for your giraffe sequences.\label{fig:paste_sequences}}
\end{center}

\vfill

\begin{center}
	\includegraphics[width=0.9\textwidth]{fig6_phyML_settings}
	\captionof{figure}{PhyML settings for the phylogenetic analysis.\label{fig:phyml}}
\end{center}

\noindent Set the substitution model to “HKY{\tablenumbers 85} (\textsc{dna/rna})” and then click the “Submit” button near the bottom of the page.

%You may have to click “Advanced Settings\dots” to access 
%some settings in this step. and then click all three
%fixed buttons. Enter 0.24 for the gamma distribution parameter, 0.46 for
%the proportion of invariant sites, and 58.4 for the
%transition/transversion ratio. Check that you have set the parameters
%correctly 

\begin{enumerate}
\item
  Wait patiently. The full workflow should take less than five minutes but might take up to ten minutes.
  You will see an animated set of \textsc{dna} sequences during the alignment
  step followed by an animated tree during the phylogenetic analysis
  step (Fig.~\ref{fig:wait}).
\end{enumerate}

\begin{center}
	\includegraphics[width=0.6\textwidth]{fig7_wait}
	\captionof{figure}{Illustration of alignment and phylogenetic analysis steps.\label{fig:wait}}
\end{center}

\begin{enumerate}
\item
  The final rendered tree will appear in the window. This type of tree is called
  a “phylogram” which shows variable branch lengths, based on number of 
  genetic differences among sequences. You will save a copy of this tree.
  
  You may also want to save your tree as a cladogram. The cladogram ignores
  branch lengths, making it easier for you to identify some clades and relationships.
  You will find the option to change from phylogram to cladogram under “Tree style:”
  near the bottom of the page.
  
  To save a copy of your phylogram, look for the the download options immediately 
  below the tree (Fig.~\ref{fig:save_tree}). Click “\textsc{pdf}” to
  save a \textsc{pdf} copy of your tree to your computer or flash drive. Name your file something
  like “giraffe\_phylogram.pdf.” Confirm that the \textsc{pdf} file has been safely saved to your 
  computer.
  
  Repeat this process if you want to save a copy of your tree as a cladogram. Be sure to
  give your file a different name, like “giraffe\_cladogram.pdf.” Again, verify the file has
  been safely saved before closing your browser window.
  
\end{enumerate}

\begin{center}
	\includegraphics[width=0.75\textwidth]{fig8_save_tree}
	\captionof{figure}{Save a copy of your tree as a \textsc{pdf} file.\label{fig:save_tree}}
\end{center}

\begin{enumerate}
\item
  Upload the \textsc{pdf} file to the ``Upload Giraffe Part 1 Files Here'' drop
  box on the course website.
\end{enumerate}

\subsection*{How to interpret your phylogenetic tree}

The red numbers on each branch are measures of confidence derived using
the approximate Log-Likelihood Ratio Test (aLRT; see Fig.~\ref{fig:phyml}), a relatively new
technique that we did not discuss in class. The aLRT derives true probabilities. The higher the
probability, the more likely the clade is the “correct” clade (based
on the data). You do not have to use 0.95 as the limit to decide whether
a clade is well supported. The authors who developed the test recommend
using between 0.75--0.85 as the lower limit of confidence. If you choose
a higher value, then you are requiring a higher probability before
accepting a clade as well supported; this is a more conservative
approach. You decide what probability level you want as part of your
decision making process.

Look for well-supported clades, where each clade contains all or nearly
all of one pattern. Look at larger clades as well as the smaller clades
within the larger clades. One or more clades will have only individuals
with the same pattern. However, one or more clades will have mostly
individuals with the same pattern but will also have at least one
individual with one a different pattern. This happens because
populations that are recently diverged will still share some of the same
alleles. However, it could also be due to low levels of gene flow. This
is \textsc{ok}. Real biological results are rarely clear-cut so we have to
interpret the results, make the final decision, and justify our
decision.

After you have come to your decision, answer the following questions. 
Upload your typed answers to the dropbox. Part~2 will be available following 
the due date for this exercise. Be sure you also upload a \textsc{pdf} of your phylogenetic tree.

%\vspace{1\baselineskip}

\subsection*{Questions}
\label{sec:questions}

\begin{questions}

\question[1]
What minimum aLRT probability value did you decide to use for
confidence?

\question[1]
Based on your analysis, how many species of giraffe have you decided
are present in Africa? 

\question[10]
Explain your reasoning for your decision. Be as clear and
precise as possible, using examples from your phylogeny. Treat this
answer as you would an essay question on an exam. I am not looking for a
single correct answer because this question has many possible answers. I want you to
apply your knowledge and proper vocabulary gained from class.

\end{questions}

\end{document}  